### DISCUSSION TITLE
Repo disabled two times

### DISCUSSION THREAD

**Discussion by anasty17** [community/community#54494] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My repository http://github.com/anasty17/mirror-leech-telegram-bot disabled 2 times. First and 2nd time for the same reason and its not true but idk what's happening, is it automated disable or someone who hit disable without checking??

1st time they enabled it but github was responding faster but now I feel like I'm talking to ghost.
If u r slow in response so try to be slow in disable and check if reasons is exists or applied on my repository!

My ticket id: 2134810

Thanks
Anas Tayyar

**Comment by anasty17** [comment]
And after reinstall it like before! Is this will happen again and again? Is it a disable game ?

**Comment by Abu3safeer** [comment]
Looks like github has some broken bots, they are just disabling repos automatically without manual checks.

**Reply by anasty17** [reply]
Mostly yes

**Comment by moviesbayx** [comment]
Such poor management of a platform like GitHub was really surprising.

**Comment by anasty17** [comment]
Repo reinstalled. I hope this will not going to happen again!

**Reply by Abu3safeer** [reply]
me too

**Comment by nandasafiqalfiansyah** [comment]
Repo reinstalled. I hope this will not going to happen again! :((
